README.md

```diff
@@ -14,7 +14,7 @@
 $ sbt new scalatra/scalatra.g8
 $ cd <name-of-app>
 $ sbt
-> Jetty/start
```

Comment:
This might require other documentation updates around the Scalatra ecosystem.

Reply:
Yeah, we will work on it.